src/genie/libs/parser/nxos/tests/test_show_platform.py

```diff
@@ -49,6 +49,7 @@ class test_show_version(unittest.TestCase):
                                  'kickstart_compile_time': '4/30/2017 23:00:00 [04/15/2017 ''04:34:05]',
                                  'kickstart_image_file': 'slot0:///n7000-s2-kickstart.10.81.0.129.gbin',
                                  'system_version': '8.1(1) [build 8.1(0.129)] [gdb]',
+                                 'version': (8, 1, 1),
```

Comment:
main purpose of parser is, to get the value as is without modification. if you need something like this, maybe having API for it would be best? like API `get_version(format='tuple')`

Reply:
This should also do the trick I believe. I might cause confusion between ```['platform']['version']``` and ```['platform']['software']['version']```

Reply:
so you will open PR for API instead?

Reply:
The API will return the version for ```['platform']['version']``` or ```['platform']['software']['version']``` ?

Reply:
@tahigash please check #386

Reply:
replied on the PR.
https://github.com/CiscoTestAutomation/genieparser/pull/386#issuecomment-770249130

Reply:
@rohit04saluja maybe we can close this PR? because you are working on API in genielibs. not in genieparser.